test/declaration_test.ts

```diff
@@ -1089,9 +1089,7 @@ declare interface X {
   new(a: string, b: number): YType;
 }
 
-declare type Y = X;
-
-declare var Y: Y;
```

Comment:
This alias doesn't work anymore since generateDartTypeName detects that Y is declared as an alias, so it ends up skipping the emission of class Y. This has been documented in #68.